pkg/resource/metadata_test.go

```diff
@@ -254,3 +255,25 @@ func TestNewMedataFromProto(t *testing.T) {
 
 	assert.True(t, md.Equal(other))
 }
+
+func BenchmarkMetadataEqual(b *testing.B) {
```

Comment:
do we have a before and after?

Reply:
please see the commit message - there's comparison there.

Reply:
sorry, missed that